furious/config.py

```diff
@@ -71,6 +71,13 @@ def get_completion_cleanup_delay():
     return config.get('cleanupdelay')
 
 
+def get_csrf_check():
+    """Get the CSRF check function, which takes one arg (webapp2.Reqeust)
+    and returns None, throwing an exception if a CSRF attack is detected.
+    """
+    return path_to_reference(get_config().get('csrf_check'))
```

Comment:
Do you need to `from furious.job_utils import path_to_reference` here like in `_get_configured_module` below? If not, I don't see where it's imported already.